src/index.js

```diff
@@ -78,3 +80,6 @@ module.exports.clearPermissions = clearPermissions
 module.exports.getPermissions = getPermissions
 module.exports.PermissionSet = PermissionSet
 module.exports.Authorization = Authorization
+
+// Export all the acl-related constants and modes at the top (index) level
+Object.assign(module.exports, aclModes.acl)
```

Comment:
Why? I would have expected the constants to be exported in some sort of `modes` or `accessModes` object to make clear that they are constants. I find `import { modes } from 'solid-permissions' ... if (someMode.equals(modes.READ)) { ... }` or `import { READ } from solid-permissions/lib/modes` clearer than `import { READ } from 'solid-permissions'`